src/frontend-admin/src/services/AxiosService/AxiosService.ts

```diff
@@ -0,0 +1,135 @@
+import axios, {AxiosInstance} from 'axios'
+//import qs from 'qs' ??
```

Comment:
This fella is super nice for creating query strings, though we _may_ not really need to do this in this project. Likely any data that we want to post to the backend will be done in the headers in one way or another.